test/python_magic_test.py

```diff
@@ -157,6 +161,8 @@ def test_descriptions(self):
         finally:
             del os.environ["TZ"]
 
+    # TODO: Fix this failing test on Ubuntu
+    @pytest.mark.skipif(sys.platform == "linux", reason="'JSON data' not found")
```

Comment:
fwiw, pytest was completely green for most linux architectures when running pytest using the cibuildwheel mechanism ref https://github.com/ahupp/python-magic/pull/294#discussion_r1590782796

Reply:
Are you able to create a targeted fix for #321?
* #321

Reply:
there was nothing to fix, pytest was green out of the box :thinking:

Reply:
maybe because im building and bundling latest libmagic from source, and github ubuntu comes with an old libmagic?